janus/lib/server/controllers/admin_controller.rb

```diff
@@ -157,6 +165,19 @@ def update_cc_agreement
       agreed: !!@params[:agreed]
     )
 
+    @user.set_guest_permissions!
+
+    if project.contact_email and @params[:agreed]
```

Comment:
With #874 , the `contact_email` can be `""`, which I think would evaluate to true in this case, but then try to send an e-mail to an empty string address? Should this do a check for `&.empty?` here, to handle that case?

Reply:
right right -- I'll check for empty here.